### ISSUE TITLE
methylqa empty plot

### ISSUE THREAD

**Issue by avilella** [ewels/MultiQC#206]
I am still getting an empty methylqa plot. I uploaded a new input data `.report` file.

**Comment by avilella** [comment]
<img width="955" alt="screen shot 2016-05-08 at 12 51 06" src="https://cloud.githubusercontent.com/assets/158007/15097694/cf6d8f2a-151b-11e6-9543-e578e1f1e984.png">

**Comment by avilella** [comment]
No apparent complains from running multiqc head master:

```
/home/avilella/.local/lib/python2.7/site-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.
  warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')
[INFO   ]         multiqc : This is MultiQC v0.7dev
[INFO   ]         multiqc : Searching '/media/sf_Downloads/sshfs/CEGX_Run184/CEGX_Run184-12345678/CEG44_16_10A-35614805/'
[INFO   ]        qualimap : Found 3 reports
[INFO   ]        methylQA : Found 3 reports
[INFO   ]          picard : Found 1 InsertSizeMetrics reports
[INFO   ]          picard : Found 1 GcBiasMetrics reports
[INFO   ]         bismark : Found 1 bismark alignment reports
[INFO   ]        cutadapt : Found 2 reports
[INFO   ]         multiqc : Report      : report_CEG44_16_10A-35614805_multiqc_dev.html
[INFO   ]         multiqc : Data        : report_CEG44_16_10A-35614805_multiqc_dev_data
[INFO   ]         multiqc : MultiQC complete
```

**Comment by ewels** [comment]
Yeah, I closed #67 because I couldn't replicate the issue. Thanks for the demo file - looks like a pretty simple case of variable space padding which I need to take into account (plus throw a warning when no data is found rather than just show an empty plot).

**Comment by ewels** [comment]
Ok, fixed. Let me know if you run into any other problems!

Phil
